correspondence/index.bs

```diff
@@ -46,4 +46,196 @@ Issue: Migrate from `APICorrespondence.md`.
 
 # Limits # {#limits}
 
-Issue: Migrate from `Limits.md`.
+This section provides references for computing WebGPU's limit values in each of the three backend APIs.
+These references inform the base values of these limits, as well as how implementations compute adapter limits.
+User agents are not required to use these formulas and may expose whatever they want
+(e.g. always choosing the base value of a limit, or bucketing devices as a fingerprinting protection).
+
+<table class=data style="font-size: 90%">
+    <thead>
+        <tr>
+            <th>Limit
+            <th>Issue
+            <th>Vulkan
+            <th>Metal
+            <th>D3D12
+        <tr>
+            <td>
+            <td>
+            <td>[Required Limits](https://www.khronos.org/registry/vulkan/specs/1.3/html/vkspec.html#limits-required)
+            <td>[Feature Set Tables](https://developer.apple.com/metal/Metal-Feature-Set-Tables.pdf)
+            <td>[Constants](https://docs.microsoft.com/en-us/windows/win32/direct3d12/constants),
+                [Hardware Tiers](https://docs.microsoft.com/en-us/windows/win32/direct3d12/hardware-support),
+                [Root Signature Limits](https://docs.microsoft.com/en-us/windows/win32/direct3d12/root-signature-limits)
+    </thead>
+    <tr>
+        <th class=issue>need a max buffer size limit
+        <td>[#1371](https://github.com/gpuweb/gpuweb/issues/1371)
+        <td>`maxMemoryAllocationSize`
+        <td>`MTLDevice.maxBufferLength`
+        <td class=issue>*No documented limit?*
```

Comment:
Nit: Noticed a strange style quirk while reviewing the preview. Any cell that is marked as an issue has a white left border, while none of the other cells do.

<img width="1456" alt="Screen Shot 2022-06-30 at 3 28 08 PM" src="https://user-images.githubusercontent.com/805273/176789161-1e658d19-fbc2-45ea-aba7-a505f3e11919.png">

Not a big deal, but it does look a little weird.

Reply:
Seems the issue class is not really intended to work with td, it just happened to look pretty nice. I'll just turn these into normal issues.